### DISCUSSION TITLE
Return an exit code if any part of scored benchmark fails

### DISCUSSION THREAD

**Discussion by fkhan6601** [aquasecurity/kube-bench#566] [Development]
Currently, we kube-bench as part of a Jenkins Job. Since the kube-bench pod does not return an exit code, the job passes regardless of the results. For CICD, it would be helpful to have an option for an exit code similar to Trivy.

**Comment by yoavrotems** [comment]
Hey I see what your point, but I think exit code should indicate on the running program, and if tests failing or passing is irrelevant to the program since the job run successfully, and all the tests ran.  
But would like to hear what you think

**Reply by fkhan6601** [reply]
I agree with @lizrice that an option to include an exit code similar to trivy would be very helpful. This would allow users to discern the difference between a job failing due to a program failure and test failure with different exit codes.

**Reply by lizrice** [reply]
I’ve raised issue #723 to track this enhancement

**Comment by lizrice** [comment]
How about we add a `--exit-code` option like Trivy, where it returns the specified error code if any tests result in FAIL?

**Comment by lizrice** [comment]
This was added with #734
